AnacondaRecipes/autovizwidget-feedstock
# autovizwidget 0.20.0

License: https://github.com/jupyter-incubator/sparkmagic/blob/master/autovizwidget/LICENSE.md
Requirements: 
- https://github.com/jupyter-incubator/sparkmagic/blob/0.20.0/autovizwidget/requirements.txt
- https://github.com/jupyter-incubator/sparkmagic/blob/0.20.0/autovizwidget/setup.py

Actions:
1. Add a license file to the recipe
2. Remove `noarch`
3. Skip `py<37`
4. Reorder `run` dependencies
5. Update `test/imports`
6. Add `pip check`
7. Update `home` url
8. Add `license_family`, `license_file`, `license_url`
9. Add `dev_url`
